tests/unit/s2n_tls13_parse_record_type_test.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+#include "testlib/s2n_testlib.h"
+#include "stuffer/s2n_stuffer.h"
+#include "tls/s2n_record.h"
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#include <s2n.h>
+
+int main(int argc, char **argv)
+{
+    uint16_t plaintext = 0xdaf3;
+    struct s2n_stuffer plaintext_stuffer = {0};
+    uint8_t record_type;
+    BEGIN_TEST();
+
+    {
```

Comment:
it's generally nice to have a comment above your test to say which case it's testing